lib/rdp-message-formatter.js

```diff
@@ -105,10 +105,25 @@ function getResponseBody(connection) {
     };
 }
 
+function getResourceTree() {
```

Comment:
When saving, file name is beaing suggested based on the current website url (e.g. crbug.com.har). Previously, current website wasn't set causing ".har" to be suggested as a file name. By responding to getResourceTree call, we can set website url to http://betwixt (causing file name suggestion to be betwixt.har)